### DISCUSSION TITLE
Only allow commands from specific roles & use/display only in certain channels.

### DISCUSSION THREAD

**Discussion by StrikerTheHedgefox** [Adikso/BashBot#18] [Ideas]
Right now I have to religiously control permissions to restrict the bot to only read and accept anything from one channel, which means that one slip-up could mean someone getting control of this thing, which isn't good. Please add the ability to make the bot only accept commands from whitelisted roles and channels.

**Comment by Adikso** [comment]
I added .whitelist command for adding user to whitelist
